zoo/services/views.py

```diff
@@ -1,22 +1,26 @@
+import json
 import re
 from collections import defaultdict
 from datetime import timedelta
 from typing import Dict, List
 
+import requests
 import structlog
 from django.core.exceptions import SuspiciousOperation
 from django.db import transaction
 from django.db.models import Q, Sum, query
-from django.http import Http404
+from django.http import Http404, JsonResponse
 from django.urls import reverse_lazy
 from django.utils import timezone
 from django.views import generic as generic_views
 from djangoql.exceptions import DjangoQLError
 from djangoql.queryset import apply_search
 
+from .models import Service
 from ..auditing.models import Issue
 from ..checklists.steps import STEPS
 from . import forms, models
+from ..repos.views import openapi_definition_from_repo
```

Comment:
I'd just leave it as `openapi_definition`. It's already clear that it's from a repository once it's on the repos module.

Reply:
...and also... there is no point anymore on having this function on `views.py`, right? as it won't be directly called via an API call anymore 🤔️

Reply:
yop was thinking about moving it to utils.py?

Reply:
sure 🙂️